raiden/transfer/token_network.py

```diff
@@ -261,38 +273,13 @@ def handle_closeroute(token_network_state, state_change):
     return TransitionResult(token_network_state, events)
 
 
-def handle_receive_transfer_refund(
```

Comment:
This function is dead code

Reply:
I already had a PR to remove this yesterday .... : https://github.com/raiden-network/raiden/pull/3532/files